engine/apps/slack/models/slack_action_record.py

```diff
@@ -1,17 +1,21 @@
 from django.db import models
 
-from apps.slack.scenarios.scenario_step import ScenarioStep
+# from apps.slack.scenarios.scenario_step import ScenarioStep
```

Comment:
can we remove this model, and any remaining references, completely? if I understand correctly, you removed the last usage of it [here](https://github.com/grafana/oncall/pull/1213/files#diff-1784f1d0d65fa477f4562e73aa23fe1c757b171f36e03f12600bdb021f121307L292).

Reply:
Yes, we can remove SlackActionRecord, but I don't want to do it in this PR, it's too big already. I prefer to do it in separate one, since migration is involved.